src/Controls/tests/DeviceTests/Elements/Image/ImageTests.cs

```diff
@@ -46,5 +47,28 @@ public async Task ImageWithUndefinedSizeAndWithBackgroundSetRenders()
 				await handler.ToPlatform().AssertContainsColor(Colors.Red, MauiContext);
 			});
 		}
+
+		[Fact]
+		public async Task ImageSetFromStreamRenders()
+		{
+			SetupBuilder();
+			var layout = new VerticalStackLayout();
+
+			var stream = File.OpenRead("red.png");
```

Comment:
This looks like it would attempt to open `/red.png` on Android and fail. This file from the other test looks like it is an `AndroidResource`.

    System.IO.FileNotFoundException : Could not find file '/red.png'.

This could only work if you added an entry to the `.csproj` as an `EmbeddedResource`, and use `Assembly.GetManifestResourceStream()`. There isn't a way to load loose files on Android, because apps remain compressed on disk.

Reply:
ah, thanks.